docs/guide/usage/integrations/home_assistant.md

```diff
@@ -39,60 +39,65 @@ The device specific configuration allows you to modify the discovery payload. He
 
 To respond to button clicks (e.g. WXKG01LM) you can use one of the following three Home Assistant configurations.
 
-### Via MQTT device trigger (recommended)
+### Via Home Assistant `event` entity (experimental)
```

Comment:
I would keep MQTT device trigger first now

Reply:
I've reversed the sections.